sum_terms.py

```diff
@@ -0,0 +1,14 @@
+"""Create series_sum function."""
+
+
```

Comment:
Consider refactoring this down.  I think you made this more complicated than it needs to be.